test/ably/realtimeconnection_test.py

```diff
@@ -41,3 +41,32 @@ async def test_auth_invalid_key(self):
             await ably.connect()
         assert ably.connection.state == ConnectionState.FAILED
         await ably.close()
+
+    async def test_connection_ping_connected(self):
+        ably = await RestSetup.get_ably_realtime()
+        await ably.connect()
+        response_time_ms = await ably.ping()
+        assert response_time_ms is not None
+        assert type(response_time_ms) is float
+
+    async def test_connection_ping_initialized(self):
```

Comment:
According to RTN13b the call to `ping` should return an exception when the client is in an invalid state so this test (and the two below) don't look correct to me